keyboards/converter/usb_usb/keymaps/bepo/keymap.c

```diff
@@ -0,0 +1,206 @@
+	M3_146,	M3_158,	M3_167,	M3_168,	M3_169,	M3_170,	M3_171,	M3_172,
+	M3_173,	M3_174,	M3_175,	M3_183,	M3_184,	M3_210,	M3_212,	M3_225,
+	M3_235,	M3_239,	M3_241,	M3_243,	M3_246,	M3_250,	M3_255,	M4_0130,
+	M4_0132,	M4_0133,	M4_0134,	M4_0135,	M4_0137,	M4_0140,
+	M4_0145,	M4_0146,	M4_0147,	M4_0148,	M4_0150,	M4_0151,
+	M4_0153,	M4_0156,	M4_8209,	M4_8239,	M4_8242,	M4_8243,
+	M4_8722,	M4_8734,	M4_8800,	M4_8804,	M4_8805,
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+	if (record->event.pressed) {
+		switch(keycode) {
+			case M1_3: WINALT1(3); break;
+			case M2_20: WINALT2(2,0); break;
+			case M2_94: WINALT2(9,4); break;
+			case M2_96: WINALT2(9,6); break;
+			case M3_126: WINALT3(1,2,6); break;
+			case M3_128: WINALT3(1,2,8); break;
+			case M3_136: WINALT3(1,3,6); break;
+			case M3_144: WINALT3(1,4,4); break;
+			case M3_145: WINALT3(1,4,5); break;
+			case M3_146: WINALT3(1,4,6); break;
+			case M3_158: WINALT3(1,5,8); break;
+			case M3_167: WINALT3(1,6,7); break;
+			case M3_168: WINALT3(1,6,8); break;
+			case M3_169: WINALT3(1,6,9); break;
+			case M3_170: WINALT3(1,7,0); break;
+			case M3_171: WINALT3(1,7,1); break;
+			case M3_172: WINALT3(1,7,2); break;
+			case M3_173: WINALT3(1,7,3); break;
+			case M3_174: WINALT3(1,7,4); break;
+			case M3_175: WINALT3(1,7,5); break;
+			case M3_183: WINALT3(1,8,3); break;
+			case M3_184: WINALT3(1,8,4); break;
+			case M3_210: WINALT3(2,1,0); break;
+			case M3_212: WINALT3(2,1,2); break;
+			case M3_225: WINALT3(2,2,5); break;
+			case M3_235: WINALT3(2,3,5); break;
+			case M3_239: WINALT3(2,3,9); break;
+			case M3_241: WINALT3(2,4,1); break;
+			case M3_243: WINALT3(2,4,3); break;
+			case M3_246: WINALT3(2,4,6); break;
+			case M3_250: WINALT3(2,5,0); break;
+			case M3_255: WINALT3(2,5,5); break;
+			case M4_0130: WINALT4(0,1,3,0); break;
+			case M4_0132: WINALT4(0,1,3,2); break;
+			case M4_0133: WINALT4(0,1,3,3); break;
+			case M4_0134: WINALT4(0,1,3,4); break;
+			case M4_0135: WINALT4(0,1,3,5); break;
+			case M4_0137: WINALT4(0,1,3,7); break;
+			case M4_0140: WINALT4(0,1,4,0); break;
+			case M4_0145: WINALT4(0,1,4,5); break;
+			case M4_0146: WINALT4(0,1,4,6); break;
+			case M4_0147: WINALT4(0,1,4,7); break;
+			case M4_0148: WINALT4(0,1,4,8); break;
+			case M4_0150: WINALT4(0,1,5,0); break;
+			case M4_0151: WINALT4(0,1,5,1); break;
+			case M4_0153: WINALT4(0,1,5,3); break;
+			case M4_0156: WINALT4(0,1,5,6); break;
+			case M4_8209: WINALT4(8,2,0,9); break;
+			case M4_8239: WINALT4(8,2,3,9); break;
+			case M4_8242: WINALT4(8,2,4,2); break;
+			case M4_8243: WINALT4(8,2,4,3); break;
+			case M4_8722: WINALT4(8,7,2,2); break;
+			case M4_8734: WINALT4(8,7,3,4); break;
+			case M4_8800: WINALT4(8,8,0,0); break;
+			case M4_8804: WINALT4(8,8,0,4); break;
+			case M4_8805: WINALT4(8,8,0,5); break;
+			case KC_RBRC: 
+				if (keyboard_report->mods & MOD_BIT(KC_RSFT) && keyboard_report->mods & MOD_BIT(KC_RALT)) {

```

Comment:
You say the user needs to use <kbd>Left Shift</kbd>+<kbd>AltGr</kbd> to activate bépo but here you check for *right* shift, not left.

Reply:
Yes, the code is not clear... indeed this one is used to switch back from azerty to bépo (the one from bépo to azerty is as a DF(4) in the layout). Nevertheless I push a change to clarify that. Thanks for the comment.